cli/auth.go

```diff
@@ -207,6 +207,55 @@ func (c *viamClient) whoAmIAction(cCtx *cli.Context) error {
 	return nil
 }
 
+// OrganizationAPIKeyCreateAction corresponds to `organization api-key create`.
+func OrganizationAPIKeyCreateAction(cCtx *cli.Context) error {
+	c, err := newViamClient(cCtx)
+	if err != nil {
+		return err
+	}
+	if err := c.ensureLoggedIn(); err != nil {
+		return err
+	}
+	orgID := cCtx.String(apiKeyCreateFlagOrgID)
+	keyName := cCtx.String(apiKeyCreateFlagName)
+	if keyName == "" {
+		// Formats name as myusername@gmail.com-2009-11-10T23:00:00Z
+		keyName = fmt.Sprintf("%s-%s", c.conf.Auth.User.Email, time.Now().Format(time.RFC3339))
+		infof(cCtx.App.Writer, "using default key name of %q", keyName)
+	}
+	resp, err := c.createOrganizationAPIKey(orgID, keyName)
+	if err != nil {
+		return err
+	}
+	infof(cCtx.App.Writer, "successfully created key:")
+	fmt.Fprintf(cCtx.App.Writer, "key id: %s\n", resp.GetId())
+	fmt.Fprintf(cCtx.App.Writer, "key value: %s\n\n", resp.GetKey())
+	warningf(cCtx.App.Writer, "keep this key somewhere safe; it has full write access to your organization")
+	return nil
+}
+
+func (c *viamClient) createOrganizationAPIKey(orgID, keyName string) (*apppb.CreateKeyResponse, error) {
+	if err := c.ensureLoggedIn(); err != nil {
```

Comment:
This `ensureLoggedIn` is probably now redundant.

Reply:
Yeah but I should not remove it. All client calls that require auth should run this.